content/en/security/application_security/enabling/nodejs.md

```diff
@@ -31,18 +31,19 @@ You can monitor application security for Node.js apps running in Docker, Kuberne
 ## Enabling threat detection
 ### Get started
 
-1. **Update your Datadog Node.js library package** to at least version 2.23.0 (for NodeJS 12+) or 3.10.0 (for NodeJS 14+), by running one of these commands:
+1. **Update your Datadog Node.js library package** to at least version 3.10.0 (for NodeJS 14+) or 4.0.0 (for Node 16+) or 5.0.0 (for Node 18+), by running one of these commands:
    ```shell
-   npm install dd-trace@^2.23.0
    npm install dd-trace@^3.10.0
+   npm install dd-trace@^4
+   npm install dd-trace@^5
    ```
    or to update from a previously installed 1.x version:
    ```shell
-   npm install dd-trace@2
+   npm install dd-trace@5
```

Comment:
Is it correct to update `dd-trace` to `v5` if I'm coming from `v1.x`?
I mean, I should update the package accordingly to NodeJS runtime version I use(`v3` for Node 14+, `v4` for Node 16+ or `v5` for Node 18+)

Reply:
Yes, this sounds better to me.